common/source/docs/common-oa-dijkstras.rst

```diff
@@ -24,6 +24,7 @@ Configuration
 
 -  :ref:`OA_TYPE <OA_TYPE>` = 2 (Dijkstra).  You may need to refresh parameters after changing this to see the parameters below.
 -  :ref:`OA_MARGIN_MAX <OA_MARGIN_MAX>`: the distance (in meters) that the vehicle should stay away from the fences and stay-out zones
+-  :ref:`OA_OPTIONS<OA_OPTIONS>` bit 2 (+4 to the value) can be set to use S-Curves around fence corners in the planned path to speed up turns. Note that using S-Curves, instead of the normal "approach,stop, turn, proceed" method of path planning around sharp fence corners, could still result in a fence breach. In those cases, this option might not be optimal.
```

Comment:
We could add a note that WPNAV_RADIUS should be set smaller than FENCE_MARGIN.  For safety waypoints should also be placed at least 10m from fence boundaries.

Reply:
done...thanks